<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::GraphicsSceneView</class>
 <widget class="QWidget" name="GammaRay::GraphicsSceneView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scene Coordinates:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sceneCoordLabel">
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Item Coordinates:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="itemCoordLabel">
       <property name="text">
        <string>0x0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GammaRay::GraphicsView" name="graphicsView">
     <property name="interactive">
      <bool>false</bool>
     </property>
     <property name="dragMode">
      <enum>QGraphicsView::ScrollHandDrag</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
